win32/Lib/win32serviceutil.py

```diff
@@ -481,8 +481,12 @@ def GetServiceClassString(cls, argv = None):
             path = os.path.split(fname)[0]
             # Eaaaahhhh - sometimes this will be a short filename, which causes
             # problems with 1.5.1 and the silly filename case rule.
-            # Get the long name
-            fname = os.path.join(path, win32api.FindFiles(fname)[0][8])
+            filelist = win32api.FindFiles(fname)
+            # win32api.FindFiles will not detect files in a zip or exe. If list is empty,
+            # skip the test and hope the file really exists. 
+            if len(filelist) is not 0:
```

Comment:
While it probably doesn't matter in practice, could you please change this to "!= 0" - using "is" for int comparisons only works due to an implementation detail.

Reply:
Well, in my opinion this is a save expression. Since len() only returns ints, I would even write
`if not len(filelist)`. Moreover, if we can be sure that win32api.FindFiles only returns lists or tuples, then `if not filelist` should be also absolutely fine.
But however, this is only my opinion.

Reply:
`not len()` is better, yes, but see https://stackoverflow.com/questions/2239737/is-it-better-to-use-is-or-for-number-comparison-in-python for an explanation - the language doesn't define `is` works for ints, it just happens to, for most ints, in cpython.

Reply:
Oh, didn't know that. Was very surprised to see that:
```
>>> -9 is -9
False
```

Reply:
I should have caught that I wanted equality comparison and not identity; thanks for the feedback and info. 
I committed the changes as requested. If there's anything else you'd like me to change, let me know.